functions/api/scheduled/recalculatePlayerAverages.ts

```diff
@@ -130,6 +133,18 @@ export const recalculatePlayerAverages = async (): Promise<void> => {
         await Promise.all(promises);
       }
     }
+    // add audit
+    const authToken = returnAuthToken(req);
+    if (!authToken) {
+      error('No admin found for audit responsible for recalculating player averages');
+    } else {
+      const userInfo = await admin.auth().verifyIdToken(authToken);
+      const auditData: Audit = {
+        admin: userInfo?.email || '',
+        description: ' recalculated player averages'
```

Comment:
forgot the timestamp field I think. Or check in the audit type for what it was called.